pkg/converter/pod_converter.go

```diff
@@ -45,6 +46,7 @@ func BuildWorkloadEndpointData(wep api.WorkloadEndpoint) WorkloadEndpointData {
 		PodName:   wep.Spec.Pod,
 		Namespace: wep.Namespace,
 		Labels:    wep.Labels,
+		FloatingIPs: wep.FloatingIPs,
```

Comment:
I think this needs to be `wep.Spec.IPNATs` - see here: https://github.com/projectcalico/libcalico-go/blob/master/lib/apis/v3/workloadendpoint.go#L55-L60